.gitignore

```diff
@@ -6,6 +6,10 @@
 target/
 bin/
 
+# Ignore generated archetypes
```

Comment:
I've spent a couple minutes looking for documentation on this, and I don't think [`archetype:create-from-project`](http://maven.apache.org/archetype/maven-archetype-plugin/create-from-project-mojo.html) has a way to support putting things in `generated-sources` to be instantiated to the user's `src` directory.